src/encoded/static/components/blocks/fallback.js

```diff
@@ -16,25 +16,56 @@ var FallbackBlockView = React.createClass({
     }
 });
 
-var FallbackBlockEdit = module.exports.FallbackBlockEdit = React.createClass({
+var JSONInput = React.createClass({
+
+    getInitialState: function() {
+        return { value: JSON.stringify(this.props.value, null, 4), error: false };
+    },
+
     render: function() {
-        var ReactForms = require('react-forms');
-        return <ReactForms.Form {...this.props} defaultValue={this.props.value} />;
+        return <div className={"rf-Field" + (this.state.error ? " rf-Field--invalid" : '')}>
+            <textarea rows="10" value={this.state.value} onChange={this.handleChange} />
+        </div>;
+    },
+
+    handleChange: function(e) {
+        var value = e.target.value;
+        this.setState({ value: value });
+        var error = false;
+        try {
+            value = JSON.parse(value);
+        } catch (e) {
+            error = true;
+        }
+        this.setState({ error });
+        if (!error) {
+            this.props.onChange(value);
+        }
     }
 });
 
+var fallbackSchema = {
+    type: 'object',
+    formInput: <JSONInput />
```

Comment:
If the form schema specifies a React element as a property's `formInput`, then the `Field` component in form.js clones that element with the necessary props.